R/executeDqChecks.R

```diff
@@ -81,6 +81,10 @@ executeDqChecks <- function(connectionDetails,
     stop("connectionDetails must be an object of class 'connectionDetails'.")
   }
 
+  if (! str_detect(cdmVersion, regex("^5.[2-4]$"))) {
```

Comment:
adding the regular expression somewhere in the package would make this easier to modify e.g. `ACCEPTED_CDM_REGEX <- "^5.[2-4]$"`

Reply:
Cool will do!